connectors/file/translator-parquet/src/main/java/org/teiid/translator/parquet/BaseParquetExecution.java

```diff
@@ -0,0 +1,341 @@
+            Comparison comparison = (Comparison) mapElement.getValue();
+            Literal l = (Literal) comparison.getRightExpression();
+            Column column;
+            Comparable value;
+            switch (comparison.getRightExpression().getType().getName()){
+                case "java.lang.String":
+                    column = FilterApi.binaryColumn(columnName);
+                    value = Binary.fromString((String) l.getValue());
+                    break;
+                case "org.teiid.core.types.BinaryType":
+                    column = FilterApi.binaryColumn(columnName);
+                    BinaryType binaryType = (BinaryType) l.getValue();
+                    value = Binary.fromConstantByteArray(binaryType.getBytes());
+                    break;
+                case "java.lang.Long":
+                    column = FilterApi.longColumn(columnName);
+                    value = (Long) l.getValue();
+                    break;
+                case "java.lang.Integer":
+                    column = FilterApi.intColumn(columnName);
+                    value = (Integer) l.getValue();
+                    break;
+                case "java.lang.Boolean":
+                    column = FilterApi.booleanColumn(columnName);
+                    value = (Boolean) l.getValue();
+                    break;
+                case "java.lang.Double":
+                    column = FilterApi.doubleColumn(columnName);
+                    value = (Double) l.getValue();
+                    break;
+                case "java.lang.Float":
+                    column =FilterApi.floatColumn(columnName);
+                    value = (Float) l.getValue();
+                    break;
+                default:
+                    throw new TranslatorException("The type " + comparison.getRightExpression().getType().getName() + " is not supported for comparison.");
+            }
+            try{
+                Method m = FilterApi.class.getMethod(getMethodName(comparison.getOperator()), Column.class, Comparable.class);
+                filterPredicate = (FilterPredicate)m.invoke(null, column, value);
+            } catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e){
+                throw new TranslatorException(e);
+            }
+            if(preFilterPredicate == null){
+                preFilterPredicate = filterPredicate;
+            }else {
+                preFilterPredicate = FilterApi.and(preFilterPredicate, filterPredicate);
+            }
+        }
+        assert filterPredicate != null;
+        return FilterCompat.get(filterPredicate);
+    }
+
+    private String getMethodName(Comparison.Operator operator) throws TranslatorException {
+        switch (operator){
+            case EQ: return "eq";
+            case GT: return "gt";
+            case LT: return "lt";
+            case GE: return "gtEq";
+            case LE: return "ltEq";
+            case NE: return "NotEq";
```

Comment:
Capitalized NotEq does not appear to be correct - can you add a not equals test?